management-server/api/proto/wso2/discovery/api/api.proto

```diff
@@ -0,0 +1,95 @@
+/*
```

Comment:
we have maintained the API including protos, protogen scripts, generated pb files in adapter side. I suppose we can maintain these also there